qiskit/aqua/algorithms/amplitude_amplifiers/grover.py

```diff
@@ -98,6 +100,13 @@ def __init__(self,
                  total number of elements in the set from the oracle used) of iterations is
                  reached. The implementation follows Section 4 of Boyer et al.
                  <https://arxiv.org/abs/quant-ph/9605034>
+            lam: For incremental search mode, the maximum number of repetition of amplitude amplification increases by factor lam,
+                 :math: `R_{i+1} = lam \times R_{i}`.
+                 Default value lam = 1.34 is proved to be optimal in
+                 <https://www.researchgate.net/publication/220133694_Grover%27s_Quantum_Algorithm_Applied_to_Global_Optimization>
```

Comment:
Could you move this part up to the class documentation, to not blow up the argument documentation?
You can add a sentence saying that 1.34 etc. is optional, and add a references block at the bottom
```
References:  
    [1]: Baritompa et al., `Title of article <link>`__
```
    [1]:

Reply:
Thank you for comment, I added the References section and I put there another reference as well.

In the class documentation, there is no description of the adaptive search and what lambda parameter is. I think we would have to extend class documentation if we want to discuss lambda there.

Can you please have a look if it's better now, or do you still want to move that part?
Thank you